lib/ext.Infobox.js

```diff
@@ -23,8 +23,10 @@ coreutil.inherits(Infobox, ExtensionHandler);
 Infobox.prototype.handleInfobox = function(manager, pipelineOpts, token, cb) {
 	// templateArgInfo is set in TemplateHandler.onTemplate
 	var arg, args = manager.frame.parentFrame.templateArgInfo[manager.frame.title].dict.params;
-	for ( arg in args ) {
-		args[arg] = args[arg].wt;
+	//TODO: it's fix for temporary fix for ticket DAT-3211. It's only symptom of bigger problem,
+	//the follow-up ticket here: DAT-3133
```

Comment:
This PR is related to https://wikia-inc.atlassian.net/browse/DAT-3211 which is about Parsoid failing to deliver response, but ticket mentioned here https://wikia-inc.atlassian.net/browse/DAT-3133 is about something else.